theme/static/js/custom.js

```diff
@@ -357,19 +403,38 @@ $(document).ready(function () {
     };
 });
 
-function showUniversalMessage(type, message, timeout) {
-    return function(response,returnType,content) {
-        if(!message) message = content;
-        if(!type) type = returnType;
-        if(!timeout) timeout = 5000;
-
-        $("#universalMessage span").html(message);
-        $("#universalMessage").attr('class','');
-        $("#universalMessage").addClass(type);
-        $("#universalMessage").slideDown();
-
-        setTimeout(function() {
-            $("#universalMessage a.um_close").click()
-        }, timeout)
+// Alert Types: "error", "success", "info"
+// pass a duration value of -1 for persistent alerts
+function customAlert(alertTitle, alertMessage, alertType, duration) {
```

Comment:
We had different methods implementing the same kind of alert functionality, so they have been combined into one method to avoid redundancy.